tests/go.mod

```diff
@@ -16,3 +16,5 @@ require (
 )
 
 replace gorm.io/gorm => ../
+
+replace gorm.io/driver/mysql => github.com/halfcrazy/mysql v1.2.3-0.20220106042630-805c93c0d8d9
```

Comment:
Can you remove those debug code, thank you.

Reply:
done

Reply:
ci get failed after switching origin MySQL driver. pls take a look at https://github.com/go-gorm/mysql/pull/60